src/Model/Command/ItemProperty.php

```diff
@@ -32,6 +32,17 @@ protected function setItemId(string $itemId): void
         $this->itemId = $itemId;
     }
 
+    protected function setProperties(array $properties): void
+    {
+        Assertion::keyNotExists(
+            $properties,
+            'item_id',
+            '$properties cannot contain key "item_id" - please use different key name.'
```

Comment:
I'm not sure "please use different key name" is the best advice.

Maybe we should rather explain the item_id is already used for identifying the item and it cannot be updated.